genet/utils/persistence.py

```diff
@@ -11,6 +12,24 @@ def ensure_dir(direc):
             logging.warning(e)
 
 
+def setify(value: Union[str, list, set]):
```

Comment:
These don't seem like persistence-related helper functions - maybe we should move them later on.

Reply:
yeah I have a really hard time naming things 🥲  what name would you give them?